Themes/PrettyFunctional (Basic).theme.json

```diff
@@ -0,0 +1,93 @@
+{
+    "name": "PrettyFunctional (Basic)",
+    "style": "Light",
+    "author": [
+        {
+            "name": "NSSynapse",
```

Comment:
You sure this is how you want to make history? I'm fine with that, just wanted to point out the forum handle/real name inconsistency and provide an opportunity to change your mind :)

Reply:
Thanks for the chance for a do-over, but I think sticking with NSSynapse makes the most sense. I probably should have used that handle on the forum, too, but in my desire to be privacy-conscious, I sometimes go a little overboard with creating different identities for different websites. ;)